src/prisma/schema.prisma

```diff
@@ -3680,3 +3682,28 @@ model ScoutProjectWalletPollEvent {
   @@index([walletId])
   @@index([toBlockNumber])
 }
+
+enum ScoutNotificationChannel {
```

Comment:
please remvoe this enum, doesnt seem to be used

Reply:
oops you are right